ga/latest/full/Dockerfile.ubi.adoptopenjdk11

```diff
@@ -21,12 +21,11 @@ RUN if [ ! -z $REPOSITORIES_PROPERTIES ]; then mkdir /opt/ibm/wlp/etc/ \
   && installUtility install --acceptLicense baseBundle \
   && if [ ! -z $REPOSITORIES_PROPERTIES ]; then rm /opt/ibm/wlp/etc/repositories.properties; fi \
   && rm -rf /output/workarea /output/logs \
-  && chmod -R g+rwx /opt/ibm/wlp/output/*
+  && find /opt/ibm/wlp ! -perm -g=rw -print0 | xargs -r -0 chmod g+rw
```

Comment:
Here, U see `x` permission is gone. It is needed to be able to open / cd into folders you need to have `x` permission.

Reply:
I don't think the installUtility touches /opt/ibm/wlp/output/* though, and if it does it looks like it is removed (workarea and logs). Do you see something in /opt/ibm/wlp/output/ that no longer has x?

Reply:
I confirmed that installUtility does not touch anything in the at /opt/ibm/wlp/output/* so we don't need to change permissions on that folder here.